knexfile.js

```diff
@@ -34,7 +35,8 @@ module.exports = {
   },
   staging: {
     client: 'pg',
-    connection: 'postgres://jbtgbgdwjucbaj:6200234cfea52892ab933ebd5d4de153976f8d0e758ec762211a921fb9718d67@ec2-174-129-255-46.compute-1.amazonaws.com:5432/d469hfan6hj3ut',
+    connection: process.env.STAGING_DB,
```

Comment:
Same as above

Reply:
Thank you for catching that, I have corrected this.